README.md

```diff
@@ -264,6 +266,10 @@ When developing, you can use the `Makefile` for doing the following operations:
 If you are working in a large organization you should learn more about [Kong
 Enterprise](https://konghq.com/kong-enterprise-edition/).
 
+## Contributions
+
+Contribution to this repository is always welcome. Please read [contributing.md](CONTRIBUTING.md) , [Code-of-Conduct.md](CODE_OF_CONDUCT.md) and [Copyrights.md](COPYRIGHT) before contributing.
```

Comment:
Short urls added. @Tieske sorry for that inconvenience. I got your suggestion wrong. Please check all the changes are done as asked by you